kubernetes/machine_classes/aws-machine-class.yaml

```diff
@@ -25,6 +25,15 @@ spec:
     namespace: default  # Namespace
     name: test-secret # Name of the secret
   blockDevices:
-    - ebs:
+    - name: /root
```

Comment:
I'm only extending the example based on multiple-volume support introduced by @rewiko in a previous change. Yes, the change was backwards-compatible. If a single volume is provided it would be treated as a the root volume and the name would be obtained from AMI (machine-class provided name ignored).

Reply:
I see, thanks.